src/web/releases.rs

```diff
@@ -579,6 +580,13 @@ pub(crate) async fn search_handler(
             return Err(AxumNope::NoResults);
         }
 
+        static RE: Lazy<Regex> = Lazy::new(|| Regex::new(r"[?&]sort=([^&]+)").unwrap());
+        if let Some(cap) = RE.captures(&query_params) {
+            sort_by = cap
+                .get(1)
+                .map_or("relevance".to_string(), |v| v.as_str().to_string());
```

Comment:
from what I see, `sort` will always be there when we have `paginate`, since the default was already applied above when generating the initial search result page, right?

Reply:
Yes, the web page use the default value(the first option) for the "sort-by".

Reply:
That means we don't need map_or here right? Because the key is always there?